sktime/forecasting/compose/_bagging.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file).
+"""Implements Bagging Forecaster."""
+
+__author__ = ["ltsaprounis"]
+
+from copy import deepcopy
+from typing import List, Union
+
+import numpy as np
+import pandas as pd
+from sklearn import clone
+from sklearn.utils import check_random_state
+from sklearn.utils._testing import set_random_state
+
+from sktime.datatypes._utilities import update_data
+from sktime.forecasting.base import BaseForecaster
+from sktime.forecasting.ets import AutoETS
+from sktime.transformations.base import BaseTransformer
+from sktime.transformations.bootstrap import STLBootstrapTransformer
+
+
+class BaggingForecaster(BaseForecaster):
+    """Forecast a time series by aggregating forecasts from its bootstraps.
+
+    Bagged "Bootstrap Aggregating" Forecasts are obtained by forecasting bootstrapped
+    time series and then aggregating the resulting forecasts. For the point forecast,
+    the different forecasts are aggregated using the mean function [1]. Prediction
+    intervals and quantiles are calculated for each time point in the forecasting
+    horizon by calculating the sampled forecast quantiles.
+
+    Bergmeir et al. (2016) [2] show that, on average, bagging ETS forecasts gives better
+    forecasts than just applying ETS directly. The default bootstraping transformer
+    and forecaster are selected as in [2].
+
+    Parameters
+    ----------
+    bootstrap_transformer : BaseTransformer (default=None)
```

Comment:
default should say the transformer instead - the function has arg `None`, but it "means" `STLBootstrapTransformer`

Reply:
It was fixed last time as well, the code snippet here is Outdated.